routes/cartPage.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://subtleui.netlify.app/styles/styles.css"
+    />
+    <script
+      src="https://kit.fontawesome.com/12af55b294.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="../styles/styles.css" />
+    <link rel="stylesheet" href="../styles/cartPage.css" />
+    <title>Sneakoholics | Cart</title>
+  </head>
+  <body>
+    <nav class="navbar fixed">
+      <a href="#" class="nav-heading">Sneakoholics</a>
+      <input
+        type="text"
+        class="input"
+        placeholder="What do you want to wear today?"
+      />
+      <ul class="nav-actions">
+        <li class="nav-action-item">
+          <button class="btn btn-primary">
+            <a href="./logout.html">Logout</a>
```

Comment:
Keeping anchor tag inside button is not a good HTML practice. You can check this https://stackoverflow.com/questions/16280684/nesting-a-inside-button-doesnt-work-in-firefox

Reply:
Thanks Aditya! Did not know this before...made the changes.